bindings/csharp/Context.cs

```diff
@@ -135,7 +164,37 @@ private static IntPtr getContextFromString(string str)
             return ptr;
         }
 
-        private Context(IntPtr ctx)
+        private static string getBackendFromUri(string uri)
+        {
```

Comment:
getContextFromString() -> returns an instance of the iio.Context class ; that's not what this does.

If you need this - you should use the C API - iio_context_get_name(ctx)

Reply:
getContextFromString() is returning a pointer to the recently created Context. This pointer is afterwards used in the Context(IntPtr ctx) constructor, to create a new instance of the iio.Context  class.

Reply:
I'm still missing things - if you want the strings - why not just call iio_context_get_name?

Reply:
I have deleted this method and the backend property from the Context class. I didn't know that backend and name are equivalent for Context. Thanks for the suggestion!